code/datums/elements/turf_transparency.dm

```diff
@@ -31,15 +32,11 @@
 	REMOVE_TRAIT(our_turf, TURF_Z_TRANSPARENT_TRAIT, ELEMENT_TRAIT(type))
 
 ///Updates the viscontents or underlays below this tile.
-/datum/element/turf_z_transparency/proc/update_multiz(turf/our_turf, prune_on_fail = FALSE, init = FALSE)
```

Comment:
@LemonInTheDark 

> I don't see any reason it couldn't be supported. Depending on how profiles work out it's something I'm considering using with #63255, and I'd like to have that option open.
> 
> Is it just a cleanliness thing? or is there a deeper issue behind it

I gutted these args because all it was doing was checking to see if the turf was openspace.  I moved these checks to `istype()` procs since IMO it was more obvious why the check was happening than passing it as an arg.  Mainly it was cleanup to improve readability.  

Since `/turf/open/openspace/space` is a subclass of `/turf/open/openspace` there shouldn't be any problems.  If you are really deadset in making this modular I can add the args back, I just don't see any benefit.

Reply:
typechecks like that smell a bit to me, specially for an element like this. IDK, it's not like other turfs couldn't in theory want that behavior, they just don't right now. 

Don't want people repeating what happen with shoving, or just sticking another typecheck in and moving on. Prefer things as clear as possible.

Maybe the vars need better documentation as to their purpose? idk

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/62875/commits/31a98491f97acd8bc04156741821ed6e95d1153e

https://github.com/tgstation/tgstation/blob/9f9f633ebd322596cccfd05ab19793ce03e46af3/code/__DEFINES/is_helpers.dm#L15-L23

The other turfs used by the helper won't ever have this element attached to it. (space, chasm, lava, water)  If this is a problem then I can make a seperate is_helper proc just for openspace turfs.  This change should also be compatible with https://github.com/tgstation/tgstation/pull/63559 as long as they add their new turf type to the is_helper list.

And just FYI, this change is untested (I'm webediting) but small enough that it shouldn't need it.